__tests__/api/getArtifactsForBranchAndWorkflow.test.ts

```diff
@@ -16,9 +16,9 @@ test('it gets workflow runs and a branch and workflow id and then gets artifacts
     owner: 'getsentry',
     repo: 'sentry',
     workflow_id: 'acceptance.yml',
-    branch: 'main',
-    per_page: 10,
-    status: 'completed',
+    // branch: 'main',
+    per_page: 30,
+    // status: 'status',
```

Comment:
We'll want to keep this status change when we revert this change. This is more correct. (`status` is overloaded in the GH API, where it can filter either a workflow run's `status` *or* the `conclusion`).